.coderabbit.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# CodeRabbit configuration
+chat:
+  auto_reply: true
+early_access: true
+reviews:
+  auto_review:
+    base_branches:
+      - "dev*"
+      - "main"
+      - "master"
+    drafts: false
+    enabled: true
+    ignore_title_keywords:
+      - "WIP"
+      - "DO NOT MERGE"
+  high_level_summary: true
+  path_filters:
```

Comment:
@coderabbitai Is specifying path_filters, files are included or excluded from the review?